deployments/deploy-tests.ps1

```diff
@@ -1,99 +1,122 @@
 try{
-    Import-Module .\azure\az-cloud-context\az-cloud-context.psd1
-    Import-Module .\azure\resources\key-vault\key-vault.psd1
-    Import-Module .\azure\resources\log-analytics\log-analytics.psd1
+
+    #shows how to import modules
     Import-Module .\powerform\pf-deployment-context\pf-deployment-context.psd1
+    Import-Module .\azure\pf-azure-context\pf-azure-context.psd1
+    Import-Module .\azure\resources\pf-key-vault\pf-key-vault.psd1
+    Import-Module .\azure\resources\pf-log-analytics\pf-log-analytics.psd1
 
-    Get-Command -Module log-analytics
-    Get-Command -Module key-vault
-    Get-Command -Module pf-deployment-context
-    Get-Command -Module az-cloud-context
+    #shows what functions are exported from each module
+    Get-Command -Module pf-key-vault
+    Get-Command -Module pf-log-analytics
+    Get-Command -Module pf-deployment-context 
+    Get-Command -Module pf-azure-context
 
     ##############################################################################################################
     #
     #               Deployment Context can drive naming defaults 
     #
     #############################################################################################################
 
-    #Still need to build this. The thought behind this is that a user wants to deploy resources out with best practices/enterprise standards
+    #The thought behind this is that a user wants to deploy resources out with best practices/enterprise standards
     #baked into the module.  By defining the deploy context parameters, the names will be created, therefore it knows
     #the RG, Sub, Resources names.  All options are going to be the defaults, so just build the definitions and deploy
 
-    #$deployContext = New-PfDeploymentContext -CompanyAbbreviation "MYAEA" -GroupAbbreviation "KKZH" -TeamAbbreviation "ADM" -EnvironmentLetter "D"
+    New-PfDeploymentContext
+
+    Set-PfAzureContext -CompanyAbbreviation "MYAEA" -GroupAbbreviation "KKZH" -TeamAbbreviation "ADM" -EnvironmentLetter "D" -Region "CentralUs"
+
+    #name, az context built from deployment context params above
+    $kv = New-PfKeyVaultBuild
```

Comment:
Is **Build** representing same "thing" as original **Definition**?

Reply:
So Build is just the build object.  My plan is that after you deploy, it will return "Definition" objects for each resource that will contain your original build object and also have cloud object within it